client/src/components/LoginSession/LoginSession.js

```diff
@@ -0,0 +1,149 @@
+/* global window */
+import React, { useState } from 'react';
+import confirm from '@silverstripe/reactstrap-confirm';
+import Config from 'lib/Config'; // eslint-disable-line
+import backend from 'lib/Backend';
+import moment from 'moment';
+import i18n from 'i18n';
+import PropTypes from 'prop-types';
+import jQuery from 'jquery';
+import 'regenerator-runtime/runtime';
+
+function LoginSession(props) {
+    const [loading, setLoading] = useState({ complete: false, failed: false, submitting: false });
+
+    async function logOut() {
+        setLoading({ ...loading, submitting: true });
+
+        const endpoint = backend.createEndpointFetcher({
+            url: `${props.LogOutEndpoint}/:id`,
+            method: 'delete',
+            payloadSchema: {
+                id: { urlReplacement: ':id', remove: true },
+                SecurityID: { querystring: true }
+            }
+        });
+
+        endpoint({
+            id: props.ID,
+            SecurityID: Config.get('SecurityID')
+        })
+            .then(response => {
+                setLoading({
+                    complete: true,
+                    failed: !!response.error && !response.success,
+                    submitting: false
+                });
+
+                if (response.success) {
+                    setTimeout(() => {
```

Comment:
Are we using this timeout to fadeout the component? If so, reactstrap has a [`<Fade />`](https://reactstrap.github.io/components/fade/) component for this purpose.

There's a good example of how to use it in [TabItem](https://github.com/silverstripe/silverstripe-admin/blob/1/client/src/components/Tabs/TabItem.js)

Reply:
I tried using this but ran into some problems with the timeout transition times not matching the unmount timeouts, so I've moved it for now: https://github.com/silverstripe/silverstripe-session-manager/issues/38